demisto_sdk/commands/common/errors.py

```diff
@@ -367,12 +370,28 @@ def invalid_v2_integration_name():
     def found_hidden_param(parameter_name):
         return f"Parameter: \"{parameter_name}\" can't be hidden. Please remove this field."
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_deprecated_integration(display_name, description):
+        error_message = ''
+        if not display_name.startswith('Deprecated.'):
```

Comment:
the test should not be here. it should be in the validator.
we should check for `(Deprecate)` at the end.